packages/platform/atoms/booker-embed/BookerEmbed.tsx

```diff
@@ -0,0 +1,11 @@
+import type { BookerPlatformWrapperAtomProps } from "../booker/BookerPlatformWrapper";
+import { BookerPlatformWrapper } from "../booker/BookerPlatformWrapper";
+import { AtomsWrapper } from "../src/components/atoms-wrapper";
+
+export const BookerEmbed = (props: BookerPlatformWrapperAtomProps) => {
```

Comment:
We call this component "BookerEmbed", but in reality it is "BookerPlatformEmbed" because this component can't be re-used for web given that it's just a "BookerPlatformWrapper" under the hood.

So I suggest:
1. In BookerEmbedPlatformWrapper.tsx move following code in place of BookerEmbed:
```
 <AtomsWrapper>
      <BookerPlatformWrapper {...props} />
    </AtomsWrapper>
  );
```
and delete this file.
2. Rename `BookerEmbedPlatformWrapper` to `BookerEmbedPlatform`.

Reply:
Maybe just rename this file to `BookerEmbedPlatformWrapper` and that's it?

Reply:
I don't see a reason for having 2 separate files when they can only be used together and either of them can't be re-used in isolation elsewhere.